.gitignore

```diff
@@ -2,3 +2,5 @@
 .Rhistory
 .RData
 .Ruserdata
+*smoke_test*
+*.json
```

Comment:
I wonder if this is too greedy of a pattern? Is there a stricter way to match the json files you want ignored?

Reply:
I think this is okay in this particular repository. All json files are auto-generated and should not be checked in.